lua/lazyvim/plugins/extras/lang/graphql.lua

```diff
@@ -0,0 +1,35 @@
+return {
+  -- add graphql to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "graphql" })
+      end
+    end,
+  },
+
+  -- correctly install lsp with mason
+  {
+    "williamboman/mason.nvim",
+    opts = function(_, opts)
+      vim.list_extend(opts.ensure_installed, {
+        "graphql-language-service-cli",
```

Comment:
Why is this needed? It wasn't needed for min 🤔 
https://github.com/Jomik/dotfiles/blob/7f709d13ae9676c76fb0ab5fa479001358ae7140/.config/nvim/lua/plugins/lang/graphql.lua#L14